server/src/main/java/com/openelements/benchscape/server/store/data/EnvironmentQuery.java

```diff
@@ -39,6 +39,9 @@ public record EnvironmentQuery(@Nullable String name,
                                @Nullable String osVersion,
                                @Nullable String jvmVersion,
                                @Nullable String jvmName,
+                               @Nullable OperationSystem osFamily,
+                               @Nullable String systemMemoryReadable,
```

Comment:
`Long systemMemory` should be used here instead of creating a new variant. If we get a "readable" String from the frontend it should be converted to a long before an `EnvironmentQuery` instance is created (in the endpoint)